xla_native_functions.yaml

```diff
@@ -76,6 +76,8 @@ supported:
   - bmm
   - cat
   - ceil
+  - celu
+  - celu_
```

Comment:
I think we don't need to lower inplace ops anymore, codegen should take care of it. Can you try to remove the `celu_` and check in `TestCeluInPlace ` whether `celu` is used instead?